cmake/DetectOptions.cmake

```diff
@@ -255,13 +255,15 @@ elseif(ADIOS2_USE_MHS)
 endif()
 
 # DataSpaces
-if(ADIOS2_USE_DataSpaces STREQUAL AUTO)
-  find_package(DataSpaces 2.1.1)
-elseif(ADIOS2_USE_DataSpaces)
-  find_package(DataSpaces 2.1.1 REQUIRED)
-endif()
-if(DATASPACES_FOUND)
-  set(ADIOS2_HAVE_DataSpaces TRUE)
+if(MPI_FOUND)
```

Comment:
@JasonRuonanWang does DataSpaces not work without MPI?
The `Engine.DataMan.DataManEngineTest.1D.Serial` test seems to suggest so.

Reply:
ah wait, I mismatched `DataMan` and `DataSpaces`

Reply:
> ah wait, I mismatched `DataMan` and `DataSpaces`.
> I guess this change in CMake could better have been a separate PR :)

In principle, yes. But cmake does not pass if DataSpaces libraries are found and MPI is disabled. This is something that must be fixed anyway in order to reproduce the error. I could have opened another PR but it would just waste another few hours before getting the problem solved. Given the amount of work I have on my desk, I'd rather compromise on these trivial things, unless someone else is willing to help separate the PRs and get them merged in a perfect way.

Reply:
Sure, sounds good. I was just confused for a sec, sorry for that :)